cluster-capi-kind/README.md

```diff
@@ -0,0 +1,5 @@
+# cluster
+
+## Description
+
+This package provides a blueprint manifest to deploy a kind cluster using cluster api
```

Comment:
It should be nice to include some defaults values in the documentation like kubernetes version, pod/service cirdrs, number of workers, etc.

Reply:
I added this:

The package contains some defaults but can be changed through the kpt pipeline
- pod cidrBlocks: 192.168.0.0/16
- service cidrBlocks: 10.128.0.0/12
- service domain: cluster.local
- kubernetes version: v1.26.3
- workers: 3

The package assumes that we mount through /opt/cni/bin the cni(s)